### PR TITLE
Press ESC twice to close Clap window

### PR DESCRIPTION
**Pull request by Avi-D-coder**
Fix: #387

### CODE REVIEW

File: ftplugin/clap_input.vim [hunk 1]
```diff
@@ -73,6 +73,8 @@ nnoremap <silent> <buffer> <CR>      :<c-u>call clap#handler#sink()<CR>
 nnoremap <silent> <buffer> <C-c>     :<c-u>call clap#handler#exit()<CR>
 nnoremap <silent> <buffer> <C-g>     :<c-u>call clap#handler#exit()<CR>
 
+nnoremap <silent> <buffer> <Esc> <Esc>:<c-u>call clap#handler#exit()<CR>
```
Comment (liuchengxu) [inline ftplugin/clap_input.vim:76]: Just `:<c-u>call clap#handler#exit()<CR>` should be enough, and please also update the doc.

Comment (Avi-D-coder) [reply ftplugin/clap_input.vim:76]: Will do, but a maybe a day before I find the time

File: README.md [hunk 2]
```diff
@@ -211,7 +211,7 @@ See `:help clap-options` for more information.
 ##### Normal mode
 
 - [x] Use <kbd>j</kbd>/<kbd>Down</kbd> or <kbd>k</kbd>/<kbd>Up</kbd> to navigate the result list up and down linewise.
-- [x] Use <kbd>Ctrl-c</kbd>, <kbd>Ctrl-g</kbd> to exit.
+- [x] Use <kbd>Ctrl-c</kbd>, <kbd>Ctrl-g</kbd> or <kbd>Esc</kbd>(vim) to exit.
```
Comment (liuchengxu) [inline README.md:214]:
```suggestion
- [x] Use <kbd>Ctrl-c</kbd>, <kbd>Ctrl-g</kbd> or <kbd>Esc</kbd> to exit.
```

**Comment by liuchengxu** [comment]
Thanks!

**Pull request merged**